pkg/alertmanager/multitenant.go

```diff
@@ -768,6 +850,14 @@ func (am *MultitenantAlertmanager) ServeHTTP(w http.ResponseWriter, req *http.Re
 	http.Error(w, "the Alertmanager is not configured", http.StatusNotFound)
 }
 
+func (am *MultitenantAlertmanager) Check(_ context.Context, _ *grpc_health_v1.HealthCheckRequest, _ ...grpc.CallOption) (*grpc_health_v1.HealthCheckResponse, error) {
```

Comment:
Why you need this? Given we have 1 global gRPC server per process, aren't we already covered by the health server registered in `pkg/cortex/cortex.go`?